### DISCUSSION TITLE
What is the purpose of the uwp triplet?

### DISCUSSION THREAD

**Discussion by brunvonlope** [microsoft/vcpkg#38558] [Q&A]
I see that vcpkg have the x64-windows and x64-uwp

How they differs? The docs says nothing

**Comment by StarGate-One** [comment]
https://learn.microsoft.com/en-us/vcpkg/users/triplets#vcpkg_cmake_system_name

**Reply by brunvonlope** [reply]
Thank you. I have read this page before opening the issue and from a build maintainer point of view, this really explains nothing.

For example: The different naming is because the Universal C runtime used or because some code in the ports is patched to don't conflict with UWP APIs?

**Comment by brunvonlope** [comment]
Just found this interesting comment: https://github.com/mstorsjo/llvm-mingw/pull/127#issuecomment-637969569

**Comment by brunvonlope** [comment]
Closing assuming as responded by https://github.com/microsoft/vcpkg/discussions/38558#discussioncomment-9314721. Would be nice more clarity on the docs by the way
